starknet_py/net/account/account.py

```diff
@@ -55,6 +55,8 @@ class Account(BaseAccount):
     Default Account implementation.
     """
 
+    ESTIMATED_FEE_MULTIPLIER: float = 1.5
```

Comment:
I wouldn't use uppercase here and perhaps add a docstring.

Reply:
https://stackoverflow.com/a/45702782

Reply:
But is it really a constant if it is intended to be changed by the user?

Reply:
It's not changeable yet though

Reply:
Well it is if one sets account.attr = 2.0